web/application/views/admin/edit-post.php

```diff
@@ -306,11 +307,180 @@ function confirmUpload() {
+                                value: null
+                            },
+                            {
+                                name: 'imageResize:50',
+                                label: '50%',
+                                value: '50'
+                            },
+                            {
+                                name: 'imageResize:75',
+                                label: '75%',
+                                value: '75'
+                            }
+                        ],
+
+                        // You need to configure the image toolbar, too, so it shows the new style
+                        // buttons as well as the resize buttons.
+                        toolbar: [
+                            'imageStyle:alignLeft', 'imageStyle:alignCenter', 'imageStyle:alignRight',
+                            '|',
+                            'imageResize',
+                            '|',
+                            'imageTextAlternative'
+                        ]
+                    },
+                    table: {
+                        contentToolbar: [
+                            'tableColumn',
+                            'tableRow',
+                            'mergeTableCells',
+                            'tableCellProperties',
+                            'tableProperties'
+                        ]
+                    },
+                    licenseKey: '',
+                    
+                } )
+                .then( editor => {
+                    console.log( Array.from( editor.ui.componentFactory.names() ) );
+                    window.editor = editor;	
+                    editor.execute( 'fontFamily', { value: 'Open Sans' } );
+                    editor.execute( 'fontSize', { value: 15 } );
+                    // editor.execute( 'ckfinder' );
+                } )
+                .catch( error => {
+                    console.error( error );
+                } );
+            //  $('#summernote_1').summernote({
+            //     height: 300,
+            //     fontSizes: ['8', '9', '10', '11', '12', '14', '18', '20', '22', '24'],
+            //     fontNames: ['Open Sans', 'Arial', 'Arial Black', 'Comic Sans MS', 'Courier New', 'Helvetica', 'Impact', 'Tahoma', 'Times New Roman', 'Verdana', 'Roboto'],
+            //     fontNamesIgnoreCheck: ['Open Sans'],
+            //     toolbar: [
+            //         ['style', ['bold', 'italic', 'underline', 'clear']],
+            //         ['font', ['strikethrough', 'superscript', 'subscript']],
+            //         ['fontsize', ['fontsize']],
+            //         ['fontname', ['fontname']],
+            //         ['color', ['color']],
+            //         ['para', ['ul', 'ol', 'paragraph']],
+            //         ['height', ['height']],
+            //         ['table', ['table']],
+            //         ['insert', ['link', 'picture', 'video', 'hr']],
+            //         ['view', ['codeview']]
```

Comment:
We need to minimise commented out code, unless it is for future use like in cases where we are disabling a feature or something.... thanks.